front/pages/api/stripe/webhook.ts

```diff
@@ -368,8 +385,15 @@ async function handler(
               status: "ended",
               endDate: new Date(),
             });
-            await revokeUsersForDowngrade(activeSubscription.workspace.id);
-            await sendOpsEmail(activeSubscription.workspace.sId);
+            const workspaceId = activeSubscription.workspace.sId;
+            const auth = await Authenticator.internalBuilderForWorkspace(
+              workspaceId,
+              true // superUser mode
```

Comment:
Not a big fan of that -- IMO this makes the method name sound a bit weird + having a dangling "true" with a comment is IMO inferior to having `{workspaceId, superUser: true}`

Reply:
As per IRL we want an `internalAdminForWorkspace` :+1:

Reply:
I actually fully agree. This will be changed anyways, this param will disappear